model/types.go

```diff
@@ -457,3 +457,9 @@ type ProcMount struct {
 	FileSystem string `json:"FileSystem,omitempty"`
 	Options    string `json:"Options,omitempty"`
 }
+
+type MountOption struct {
+        UserSquashing string
+        NfsVersion    string
+        ReadOnly      bool
+}
```

Comment:
`UserSquashing` should be handled by the CSP parameters.
`NfsVersion` should server side should be handled by the CSP parameters, client side in the StorageClass mountOptions.
`ReadOnly` should be handled by the CSP parameters.

Please close this PR.